### PR TITLE
Release/coppercrm merge

### PR DESCRIPTION
**Pull request by timliuhull**
reviewed Copper, Warehouse and Outreach updates.

### CODE REVIEW

File: packages/connectors/hull-pipedrive/package.json [hunk 1]
```diff
@@ -1,10 +1,9 @@
 {
   "name": "hull-pipedrive",
-  "version": "0.1.17-monorepo",
+  "version": "0.1.18-monorepo",
   "description": "Sync persons and organizations with Hull",
   "scripts": {
-    "test": "yarn run test:integration",
-    "test:integration": "DEBUG=hull* NODE_ENV=test mocha --exit --require ../../../root-babel-register.js -R spec ./test/integration/{*.js,**/*.js}",
+    "test": "exit 0; echo 'tests are run by global jest'",
```
Comment (timliuhull) [inline packages/connectors/hull-pipedrive/package.json]: I think this is right... should be running pipedrive tests with jest

File: packages/hull-client/src/client.js [hunk 2]
```diff
@@ -199,13 +234,45 @@ class HullClient {
...
-        logsArray.push({
-          message,
-          level,
-          data: payload.data,
-          context: payload.context,
-          timestamp: new Date().toISOString()
-        });
+        let transformedLog;
+
+        if (this.config.esLogTransform) {
```
Comment (timliuhull) [inline packages/hull-client/src/client.js]: @andyh565 This look right?

File: packages/connectors/hull-pipedrive/package.json (hunk 1 above)
Comment (louisjahn) [reply packages/connectors/hull-pipedrive/package.json]: Yes they are!

File: packages/hull-client/src/client.js (hunk 2 above)
Comment (andyh565) [reply packages/hull-client/src/client.js]: yeah if you're good with it

**Pull request merged**